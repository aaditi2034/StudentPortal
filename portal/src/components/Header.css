.form-container{
    margin: 0 auto;
    width:50%;
}

.err {
    color: red;
}

.btn {
    margin-left: 5px;
}