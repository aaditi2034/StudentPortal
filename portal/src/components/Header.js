import React from 'react';
import './Header.css';

const Header = () => (
  <nav className="navbar navbar-expand-sm bg-info">
    <div className="navbar-brand">STUDENT PORTAL</div>
  </nav>
);

export default Header;
