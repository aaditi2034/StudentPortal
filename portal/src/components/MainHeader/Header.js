import React from 'react';
import './Header.css';

const Header = () => (
  <nav className="navbar navbar-expand-sm bg-info">
    <div className="navbar-brand fancy-h1">Student Portal</div>
  </nav>
);

export default Header;
