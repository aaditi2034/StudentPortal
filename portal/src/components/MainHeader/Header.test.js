/* eslint-disable no-undef */
import React from 'react';
import {
  render,
  cleanup,
} from 'react-testing-library';
import 'jest-dom/extend-expect';
import Header from './Header';

afterEach(cleanup);

it('renders', () => {
  const { asFragment } = render(<Header text="Student Portal" />);
  expect(asFragment()).toMatchSnapshot();
});

it('inserts text in h1', () => {
  const { getByText } = render(<Header text="Student Portal" />);
  expect(getByText('Student Portal')).toHaveClass('fancy-h1');
});
