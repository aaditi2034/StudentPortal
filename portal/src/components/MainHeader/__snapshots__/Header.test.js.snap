// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <nav
    class="navbar navbar-expand-sm bg-info"
  >
    <div
      class="navbar-brand fancy-h1"
    >
      Student Portal
    </div>
  </nav>
</DocumentFragment>
`;
